test/parallel/test-whatwg-url-properties.js

```diff
@@ -1,9 +1,11 @@
+// Flags: --expose-internals
 'use strict';
 
 require('../common');
 
 const URL = require('url').URL;
 const assert = require('assert');
+const urlToOptions = require('internal/url').urlToOptions;
```

Comment:
Maybe this one deserves a separate test? The name of this test doesn't really match this API here.